src/block.rs

```diff
@@ -159,14 +179,34 @@ impl BlockBuilder {
 
 #[cfg(test)]
 mod tests {
+    use crate::test_utils::{gen_attrs, gen_empty_attrs};
+
     use super::*;
-    use crate::test_utils::gen_attrs;
 
     #[test]
     fn test_block() {
         let mut builder = BlockBuilder::new(4096, vec![RowFeature::Flags]);
-        assert!(builder.add(b"key1", Some(b"value1"), gen_attrs(0)));
-        assert!(builder.add(b"key2", Some(b"value2"), gen_attrs(1)));
+        assert!(builder.add(RowEntry::new(
```

Comment:
nit: can these use `add_kv`?

Reply:
yes indeed, the style would be more consistent with `add_kv` here, fixed in 11dcc99